Assets/Mirror/Runtime/NetworkConnection.cs

```diff
@@ -54,7 +54,7 @@ public class NetworkConnection : IDisposable
         /// The IP address / URL / FQDN associated with the connection.
         /// Can be useful for a game master to do IP Bans etc.
         /// </summary>
-        public string address;
+        public abstract string address { get; }
```

Comment:
why do we still have an address field here?

Reply:
Done.  I would love to get rid of all the castings,  but it would mean changing  a lot of method signatures,   I already went through that rabbit hole and it ends up breaking too much.

Reply:
maybe get rid of conn.address in callers too then? that's what I meant before, sorry.
we don't really need it in networkpreview there, and that one debug.log isn't that useful either.
if someone needs address they can use transport

Reply:
if someone is currently using it,  this will break their game no?

I am fine removing it,  but maybe as a separate PR?